scripts/execute-deepnote.py

```
@@ -0,0 +1,891 @@
+                    except Exception as e:
+                        print(f"    ⚠ Communication error: {e}")
+                        break
+                
+                if not had_output and not had_error:
+                    print("    (no output)")
+                
+                print()  # Empty line between blocks
+        
+        # Summary
+        print(f"{'='*70}")
+        total_blocks = len(code_blocks) + len(sql_blocks) + len(chart_blocks)
```

Ruff (0.14.3)

Local variable `total_blocks` is assigned to but never used

Remove assignment to unused variable `total_blocks`

(F841)